test/functionnal-tests/README.md

```diff
@@ -0,0 +1,15 @@
+#FUNCTIONNAL TEST
+
+##What ?
+Functionnal tests are unit tests that ensure the functionment of the parameter-framework.
```

Comment:
be careful: functionnal test are *not* unit tests. They are conceptually different